pytorch_blade/torch_blade/exporter.py

```diff
@@ -241,7 +246,12 @@ def replace_module_with_trace(mod, tracing_list, prefix=''):
     if isinstance(model, nn.DataParallel) or isinstance(model, nn.parallel.DistributedDataParallel):
         model = model.module
 
-    _model = _deepcopy(model)
+    with Config.get_current_context_or_new() as cfg:
+        # If disable_optimization_for_inference is True, we will not need deep copy of the model.
```

Comment:
Why？

Another question: How could training workflow with Dynamo reach here ? The export function seems to deal with `nn.Module` only.

Reply:
torch_blade uses `torch_blade.optimize` API to optimize the torchscript as Dynamo backend.